src/main/java/org/mockbukkit/mockbukkit/world/WorldMock.java

```diff
@@ -2928,4 +2928,9 @@ public void playSound(@NotNull Location location, @NotNull Sound sound, @NotNull
 		throw new UnimplementedOperationException();
 	}
 
+	public void tick()
+	{
+		fullTime++;
```

Comment:
Not sure, but shouldn't this change the gametime as well?

Reply:
True, however, I'm wondering what the difference between these 2 is. Because now I'm just doing ++ for both?